.pre-commit-config.yaml

```diff
@@ -40,7 +40,7 @@ repos:
           - --ignore=E203,W503
           - --max-complexity=25
           - --max-line-length=88
-        additional_dependencies: [pep8-naming]
+        additional_dependencies: [pep8-naming, flake8-builtins]
```

Comment:
Let's put these in alphabetical order.  When lists are short it is not a problem but when lists are long it makes spotting missing elements easier and makes inserting duplicate elements harder.  Lists that start out unsorted usually end up unsorted so it is a good idea to start sorting early.

Reply:
> Let's put these in alphabetical order. When lists are short it is not a problem but when lists are long it makes spotting missing elements easier and makes inserting duplicate elements harder. Lists that start out unsorted usually end up unsorted so it is a good idea to start sorting early.

Good point